sc2/game_data.py

```diff
@@ -258,27 +256,6 @@ def cost(self) -> "Cost":
         return Cost(self._proto.mineral_cost, self._proto.vespene_cost, self._proto.research_time)
 
 
-class EffectData:
```

Comment:
Why is EffectData removed?

Reply:
Note that this is currently not working because the client is not updated yet.
With this protocol update, the effect radius, alliance and owner will be populated by `gamestate` once the client is updated and actually sends this. See [here](https://github.com/Blizzard/s2client-proto/pull/135/files#diff-3b331650a4f7c9271a579b31cf771ed5R146).
Until now, I used the data in `gamedata` to complete the effect data that was missing like the radius.
Now that I think about this, maybe we should keep it this way, because the radius will always be the same size for each effect type.

Reply:
I'm not sure. The Linux client might take forever to update. Waiting for the retail client updatte might be a smart thing to do.